locations/hours.py

```diff
@@ -1093,6 +1134,53 @@ def hours_extraction_regex(
         )
         return full_regex
 
+    @staticmethod
+    def closed_days_extraction_regex(
```

Comment:
I'd like to remove this code duplication here. Can you find a better way to do this?